css-transitions-2/Overview.bs

```diff
@@ -80,6 +80,16 @@ sub-property that is animatable. The <dfn>expanded transition property
 name</dfn> of a transition is the name of the longhand sub-property for which
 the transition was generated (e.g. 'border-left-width').
 
+## The transition timing function ## {#teransition-timing-function}
+
+When creating a {{KeyframeEffect}} for a transition, the effect's {{EffectTiming/easing}}
+option is set to the value of the matching 'transition-timing-function' property,
+which must be used by {{AnimationEffect/getComputedTiming()}}.
+
+Note, this behavior was chosen over the {{CSSAnimation}} behavior of setting
```

Comment:
If we use `Note:` instead, does this get formatted as a note?

Reply:
Also, rather than referring to `CSSAnimation`, it might make more sense to just refer to CSS Animations. e.g.

```
Note: Unlike CSS animations, the timing function for CSS transitions applies to the animation effect as opposed to the individual keyframes since this allows it to be reflected in the [=transformed progress=] as used when calculating the reversing shortening factor.
```

Reply:
done. Thanks for the reword.